core/spec/models/spree/store_spec.rb

```diff
@@ -37,19 +37,32 @@
   end
 
   describe ".default" do
-    let!(:store)    { create(:store) }
-    let!(:store_2)  { create(:store, default: true) }
+    before do
+      Spree::Store.delete_all
```

Comment:
yep, I thought it might be, but it's so easy to mess this test up so it's not testing what you want (as it was before), that I thought it good to put in the explicit delete to make absolutely sure it's testing what it's supposed to. 

Especially because it requires macro knowledge of what's going on in the test suite and solidus code (some code that at some point has lazily created a default store?) to be _sure_ there aren't any stores created before.  Truncating at the start of the test suite isn't enough, you also need to be certain you have something running before each test that makes sure the db is being reset. Which there should be -- but I still don't understand how all the _other_ tests that require a default store existing actually work in that case!  

There is so much macro knowledge you need to be sure, and so many things that could change to make it not so, and such risk of the test not testing what is intended,  it just seemed safer to explicitly enforce the expectations instead of just counting on everything else working how you expect. 

But if you still don't after this persuasive essay, I'll get rid of it. Let me know?